awscli/shorthand.py

```diff
@@ -91,10 +94,12 @@ def __init__(self, value, expected, actual, index):
         super(ShorthandParseSyntaxError, self).__init__(msg)
 
     def _construct_msg(self):
+        expected_txt = ' or '.join(self.expected) \
```

Comment:
For reviewer, do we suspect this could potentially be a breaking change? We are modifying the text of an exception that gets thrown on incorrect shorthand syntax input. 

This felt like a good solution because there are now cases in which we must expect one of multiple substrings while parsing shorthand syntax. For example, after parsing the key of a key-value pair, the parser now expects either '=' or '@='. If the user instead supplied neither, I think it is useful to enumerate all of the correct possible operators.